packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -22,26 +32,50 @@ export interface DescriptionListProps extends Omit<React.HTMLProps<HTMLDListElem
     xl?: '1Col' | '2Col' | '3Col';
     '2xl'?: '1Col' | '2Col' | '3Col';
   };
+  autoFitModifier?: {
+    default?: string;
+    md?: string;
+    lg?: string;
+    xl?: string;
+    '2xl'?: string;
+  };
 }
 
+const setAutoFitModifiers = (autoFitModifier: AutoFitModifiers) => {
+  const prefix = '--pf-c-description-list--GridTemplateColumns--min';
+  const mods = autoFitModifier as Partial<{ [k: string]: string }>;
+  return Object.keys(mods || {}).reduce(
+    (acc, curr) =>
+      curr === 'default' ? { ...acc, [prefix]: mods[curr] } : { ...acc, [`${prefix}-on-${curr}`]: mods[curr] },
+    {}
+  );
+};
+
 export const DescriptionList: React.FunctionComponent<DescriptionListProps> = ({
   className = '',
   children = null,
   isHorizontal = false,
   isAutoColumnWidths,
+  isAutoFit,
   isInlineGrid,
   columnModifier,
+  autoFitModifier = undefined,
```

Comment:
I would probably include `Min` here, as we might need to add a `-max` value in the future.

Reply:
`autoMinFitModifier` or `autoFitMinModifier` or maybe `autoFitModifierMin`? :laughing:

Reply:
I would say `autoFitMinModifier`.

Reply:
Got it! :grinning:  Thanks